### PR TITLE
added the different trust layers more explicitly

### PR DESCRIPTION
**Pull request by woutslakhorst**

### CODE REVIEW

File: rfc/rfc001-nuts-start-architecture.md [hunk 1]
```diff
@@ -114,10 +114,20 @@ In a perfect world, the legal base should also have a digital signature that can
...
+
+The Nuts trust model is multi-layered. All layers combined provide the level of trust that is required for exchanging medical data. Each layer has its own set of allowed digital signatures. The separation makes sure an organisation can't act as a care professional and that a service provider can't act as a care organisation. The exception is a one-man practise where the care professional is also the owner of the organisation. 
+
+| layer     | responsible party | establishes trust in  |
+|-------    |----------------   |--------------------	|
+| user 	    | Care professional | a user session exists. This session is linked to the care organisation. [\[RFC002\]](rfc002-authentication-token.md).	|
+| data 	    | Care organisation	| registry data. The connection between care organisations, services and endpoints. \[RFC00X?\]. |
```
Comment (reinkrul) [inline rfc/rfc001-nuts-start-architecture.md:124]: This is RFC006

File: rfc/rfc001-nuts-start-architecture.md [hunk 2]
```diff
@@ -114,10 +114,20 @@ In a perfect world, the legal base should also have a digital signature that can
...
+| network 	| Service provider 	| members of the private network. \[RFC00X?\].	|
```
Comment (reinkrul) [inline rfc/rfc001-nuts-start-architecture.md:125]: This is going to be RFC005

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: rfc/rfc001-nuts-start-architecture.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply rfc/rfc001-nuts-start-architecture.md:125]: rfc001 doesn't talk about vendors....

File: rfc/rfc001-nuts-start-architecture.md [hunk 3]
```diff
@@ -114,10 +114,20 @@ In a perfect world, the legal base should also have a digital signature that can
...
+Trust is, in the case of Nuts, the collective of authorizations, digital signatures and curated list of trusted 3rd parties. This relates closely to the general security of the network as a whole. An exchange of data between a custodian and actor is only allowed if all involved parties can be trusted \[M7\]. An action in the virtual domain must be linkable to real world parties \[M6\]. 
+
+The Nuts trust model is multi-layered. All layers combined provide the level of trust that is required for exchanging medical data. Each layer has its own set of allowed digital signatures. The separation makes sure an organisation can't act as a care professional and that a service provider can't act as a care organisation. The exception is a one-man practise where the care professional is also the owner of the organisation.
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply rfc/rfc001-nuts-start-architecture.md:119]: Added on-premise to the exception list

File: rfc/rfc001-nuts-start-architecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply rfc/rfc001-nuts-start-architecture.md:124]: Via digital signatures originating from a source held by the care organisation.

**Review by reinkrul — APPROVED** [review]

**Pull request merged**